components/header.tsx

```
@@ -96,44 +99,48 @@ const Header: React.FC = () => {
                 </button>
               </VeltSidebarButton>
 
               {/* Theme Toggle */}
               <div
-                className={`${styles['theme-toggle']} ${isDarkMode ? styles['dark'] : styles['light']}`}
+                className={`${styles["theme-toggle"]} ${
+                  isDarkMode ? styles["dark"] : styles["light"]
+                }`}
                 onClick={toggleDarkMode}
                 title="Toggle dark mode"
               >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)